### ISSUE TITLE
Union is unused

### ISSUE THREAD

**Issue by srghma** [purescript-codegen/purescript-ps-ast#2]
can we remove 

https://github.com/purescript-codegen/purescript-ps-ast/blob/bcc843d867c8328ee9cd16d735f8e91133d15c2d/src/Language/PS/AST/Types.purs#L146-L163

?

**Comment by paluh** [comment]
Yeah. .Thanks for pointing that out. This lib was literally copied from mui and I was not able to clean it up :-)
